### PR TITLE
Added a feature to add courses to playlist of the user

### PR DESCRIPTION
**Pull request by Divyanshgarg-04**
Added a feature to add courses to playlist of the user

### CODE REVIEW

**Comment by Divyanshgarg-04** [comment]
@anand-harsh Kindly check the changes and if any other changes are required please let me know

**Comment by anand-harsh** [comment]
Share me screenshot of the response and url you using, means full screenshot of postman

**Comment by Divyanshgarg-04** [comment]
courses present
image

Course already in the playlist
image


For new course
image

Adding the new course to playlist
image

**Comment by Divyanshgarg-04** [comment]
@anand-harsh  any suggestions?

File: controllers/userController.js [hunk 1]
```diff
@@ -256,3 +256,40 @@ export const updateUserRole = async (req, res, next) => {
     next(new ErrorHandler("Error updating user role", 400));
   }
 };
+
+export const addCourse = catchAsyncError (async(req, res, next) => {
```
Comment (anand-harsh) [inline controllers/userController.js:260]: rename component to "addCourseToUserPlaylist"

**Review by anand-harsh — CHANGES_REQUESTED** [review]
Also update your branch and all good to go

**Pull request closed without merging**